DEV/org.openl.rules/src/org/openl/rules/lang/xls/XlsSheetSourceCodeModule.java

```diff
@@ -62,16 +62,25 @@ public int getStartPosition() {
         return 0;
     }
 
+    @Override
+    public String getFileUri() {
+        return toSheetUri(workbookSource.getFileUri());

```

Comment:
Where is it used? File URI is a path to a file and there are no needs to define sheet location.
Remove toSheetUri() wrapping.